tests/suites/cloud/tests/ssh-auth/index.js

```diff
@@ -117,26 +117,25 @@ module.exports = {
 					}).catch((err) => {
 						return test.match(
 							err.message,
-							/All configured authentication methods failed/,
+							/All configured authentication methods failed|Connection lost before handshake/,
 							"Local SSH authentication without custom keys is not allowed in production mode"
 						);
 					});
 				}).then(async () => {
-					return exec(`ssh-add ${sshPath}`)
-					.then(() => {
-						this.worker.addSSHKey(sshPath);
-					})
-					.then(() => {
-						return setConfig(test, this, this.balena.uuid, 'os.sshKeys', [customKey.pubKey.trim()]);
-					});
-				}).then(async () => {
+					await exec(`ssh-add ${sshPath}`);
+					await this.worker.addSSHKey(sshPath);
+				})
+				.then(() => {
+					return setConfig(test, this, this.balena.uuid, 'os.sshKeys', [customKey.pubKey.trim()]);
+				})
+				.then(async () => {
```

Comment:
What was the reason for this change? It looks functionally the same to me.

Reply:
true - this is left over from when I was trying something out - will revert